composer.json

```diff
@@ -27,12 +27,12 @@
     "require": {
         "php": "^7.2|^8.0",
```

Comment:
Min. required PHP version for Laravel 8 is PHP 7.3, maybe all the illuminate packages have the same requirement